### PR TITLE
Add kaldi

### PR DESCRIPTION
**Pull request by mcernak**
This aims to be a binary distribution of the Kaldi speech recognition framework.

### CODE REVIEW

**Comment by mcernak** [comment]
@183amir The kaldi build fails on speex dependency .. I probably need to wain until speex package is built.

**Comment by 183amir** [comment]
yeah I was suspecting that. Maybe you can rename the `speex` folder to something like `speex1` for now.

**Comment by mcernak** [comment]
@183amir The tests passed, can we try to merge it?

File: recipes/speex1/meta.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-{% set version = "1.2rc2" %}
```
Comment (jakirkham) [inline recipes/speex1/meta.yaml]: Did you intend to move this directory? That might mess some things up.

File: recipes/speex1/build.sh~ [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```
Comment (jakirkham) [inline recipes/speex1/build.sh~]: Is this file suppose to be here? It has `~` after its name. 😕

**Comment by mcernak** [comment]
@jakirkham speex1 moved back to speex, and build.sh~ is removed - can I do something more to resolve the conflict?

**Comment by mcernak** [comment]
@183amir What can I do here? The tests passed, this conflict appeared by renaming speex1 to speex, as requested.

**Comment by 183amir** [comment]
Could you rebase on master and squash please?

**Comment by 183amir** [comment]
please rename the title of merge request too.

**Comment by ericdill** [comment]
Closing this in favor of #2727

**Pull request closed without merging**